apps/backoffice-v2/src/domains/business-reports/hooks/mutations/useCreateBusinessReportBatchMutation/useCreateBusinessReportBatchMutation.tsx

```diff
@@ -29,9 +29,6 @@ export const useCreateBusinessReportBatchMutation = ({
         merchantSheet,
         isExample: customer?.config?.isExample ?? false,
       });
-
-      // Artificial delay to ensure report is created in Unified API's DB
-      await new Promise(resolve => setTimeout(resolve, 3000));
```

Comment:
No longer necessary, as we create the report immediately and not in the workflow.